dashboard/data.py

```diff
@@ -13,25 +13,27 @@
 def get_connection() -> connection:
     """Get database connection to the PostgreSQL database."""
     return psycopg2.connect(
-        host=ENV("DB_HOST"),
-        dbname=ENV("DB_NAME"),
-        user=ENV("DB_USER"),
-        password=ENV("DB_PASSWORD"),
-        port=ENV("DB_PORT")
+        host=ENV["HOST"],
+        dbname=ENV["DATABASE_NAME"],
+        user=ENV["DATABASE_USERNAME"],
+        password=ENV["DATABASE_PASSWORD"],
+        port=ENV["PORT"]
     )
 
 
 def execute_query(query: str, params=None) -> pd.DataFrame:
     """Execute query and handle the connection/cursor."""
     conn = get_connection()
+    if params is not None and not isinstance(params, (list, tuple)):
```

Comment:
I think you can just do 
```suggestion
    if params and not isinstance(params, (list, tuple)):
```

because that's basically just a double negative